### PR TITLE
SDK links and icons

### PR DESCRIPTION
**Pull request by sacOO7**
Added code to update latest SDK links and respective icons
Optimized current background video for fast loading

### CODE REVIEW

File: public/app/views/index.html [hunk 1]
```diff
@@ -130,7 +146,7 @@ <h3 class="home-sponsored-by-heading">PROUDLY SPONSORED BY</h3>
             <a href="https://lisk.io/" target="_blank" style="width: 180px">
                 <img style="width: 180px" src="../assets/img/lisk_logo_text.png">
             </a>
-         </div>
+         </div>S
```
Comment (MegaGM) [inline public/app/views/index.html]: what's that?

Comment (jondubois) [reply public/app/views/index.html]: Well spotted @MegaGM

**Comment by jondubois** [comment]
@sacOO7 I'll have a look at this PR later in the week and merge/deploy then so feel free to make adjustments in the meantime if necessary.

**Comment by sacOO7** [comment]
Hi @MegaGM Thank you for the suggestion. Actually it was present in the first commit, but I fixed in second commit. So, I don't think the change that you requested will be present in latest commit of PR.

**Comment by sacOO7** [comment]
hi @jondubois , can you merge this PR as soon as possible. It might look big, but most of the changes are indentation based. It will take just around 5 mins to review it. Thank you once again :)

**Comment by jondubois** [comment]
@sacOO7 Thanks, looks great! Sorry for the delay I've been very busy recently.
Thanks @MegaGM for doing peer review :)

**Pull request merged**